core/src/main/java/org/testcontainers/containers/image/ImageData.java

```diff
@@ -0,0 +1,34 @@
+package org.testcontainers.containers.image;
+
+import com.github.dockerjava.api.command.InspectImageResponse;
+import com.github.dockerjava.api.model.Image;
+import java.time.OffsetDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+@Data
+@RequiredArgsConstructor(access = AccessLevel.MODULE)
+public class ImageData {
+
+    public static ImageData from(Image image) {
+        return new ImageData(image.getCreated(), image.getId(), image.getParentId(), Arrays.asList(image.getRepoTags()), new ArrayList<>(), image.getSize(), image.getVirtualSize());
+    }
+
+    public static ImageData from(InspectImageResponse imageResponse) {
+        return new ImageData(OffsetDateTime.parse(imageResponse.getCreated()).toEpochSecond(), imageResponse.getId(), imageResponse.getParent(), imageResponse.getRepoTags(), imageResponse.getRepoDigests(), imageResponse.getSize(), imageResponse.getVirtualSize());
+    }
+
+    private final Long created;
```

Comment:
FYI if you annotate with `@Value` (and it is a value), you can drop `private final` since it is so by default